recipes/pennylane-lightning-gpu/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "pennylane-lightning-gpu" %}
+{% set version = "0.29.0" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  # - url: https://github.com/PennyLaneAI/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  #   sha256: e4cb8c496bddbb6c6cd27a6e1462cd3625952996222185dd0d7ec82e3cafea9b
+  - git_url: https://github.com/PennyLaneAI/pennylane-lightning-gpu.git
+    git_branch: bugfix/CUSPARSE_MV_ALG_DEFAULT
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version !=  "11.2")]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  # script: export LD_LIBRARY_PATH={{ PREFIX }}/lib && {{ PYTHON }} -m pip install . --no-deps -vv #https://github.com/introlab/rtabmap_ros/issues/131
+  missing_dso_whitelist:
+    - '*/libcuda.*'  # [linux]
+    - '*/libcustatevec.*'  # [linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake
+    - ninja
+  host:
+    - {{ pin_compatible('cudatoolkit', max_pin='x', lower_bound='11.2') }} # cudatoolkit >=11.2,<12.0.0
+    - cuquantum
+    - libgomp      # [linux]
+    - llvm-openmp  # [osx]
+    - pip
+    - pybind11
+    - python
+  run:
+    - {{ pin_compatible('cudatoolkit', max_pin='x', lower_bound='11.2') }} # cudatoolkit >=11.2,<12.0.0
+    - cuquantum
+    - pennylane
+    - pip
```

Comment:
nit: I don't think you need `pip` at runtime?
```suggestion
```

Reply:
It's not really required in practice, but it is used in some functions as in 
```
python -c "import pennylane as qml; qml.about()"
```

Reply:
Can you explain more? You already have pip listed in `test/requries`, why does this package also depend on pip at runtime?